frontend/src/metabase/entities/databases/big-query-fields.js

```diff
@@ -113,17 +116,63 @@ export default function getFieldsForBigQuery(details) {
               required: true,
             },
           ]),
+      {
+        name: "advanced-options",
+        type: "section",
+        default: false,
+      },
       {
         name: "use-jvm-timezone",
         "display-name": t`Use the Java Virtual Machine (JVM) timezone`,
         default: false,
         type: "boolean",
+        "visible-if": { "advanced-options": true },
       },
       {
         name: "include-user-id-and-hash",
         "display-name": t`Include User ID and query hash in queries`,
         default: true,
         type: "boolean",
+        "visible-if": { "advanced-options": true },
+      },
+      {
+        name: "auto_run_queries",
+        type: "boolean",
```

Comment:
It seems strange that these descriptions for generic database settings are in `big-query-fields.js`

Reply:
I agree, however these are just the field definitions for the deprecated driver, so they will be removed soon. Since the generic setting definitions are being moved to the backend in this PR, we also have to add them here for the time being.